tests/test_rpc.py

```diff
@@ -1348,3 +1348,317 @@ def test_gethistory(revault_network, bitcoind, executor):
+    )
+
+    wait_for(lambda: len(bitcoind.rpc.getrawmempool()) == 3)
+    for spend_txid in spend_txids:
+        entry = bitcoind.rpc.getmempoolentry(spend_txid)
+        assert entry["descendantcount"] == 2
+        package_feerate = entry["fees"]["descendant"] * COIN / entry["descendantsize"]
+        assert package_feerate >= 50
+
+
+## Sanity checks
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_manual_cpfp_invalid_txid(revault_network, bitcoind):
+    CSV = 12
+    revault_network.deploy(
+        2,
+        1,
+        csv=CSV,
+        bitcoind_rpc_mocks={"estimatesmartfee": {"feerate": 0.0005}},  # 50 sats/vbyte
+    )
+    man = revault_network.mans()[0]
+    # removed one character 4
+    txid = "808cd1c4cd6d0d3abbfd5c47bb4869dbc25ffcb1dcd5effdef31396c17a68cf"
+    with pytest.raises(RpcError) as err:
+        man.rpc.cpfp([txid], 50)
+
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_manual_cpfp_unknown_txid(revault_network, bitcoind):
+    CSV = 12
+    revault_network.deploy(
+        2,
+        1,
+        csv=CSV,
+        bitcoind_rpc_mocks={"estimatesmartfee": {"feerate": 0.0005}},  # 50 sats/vbyte
+    )
+    man = revault_network.mans()[0]
+
+    txid = "4808cd1c4cd6d0d3abbfd5c47bb4869dbc25ffcb1dcd5effdef31396c17a68cf"
+    # unknown transaction should throw an error.
+    with pytest.raises(RpcError) as err:
+        man.rpc.cpfp([txid], 50)
+
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_manual_cpfp_already_mined(revault_network, bitcoind):
+    CSV = 12
+    revault_network.deploy(
+        2,
+        1,
+        csv=CSV,
+        bitcoind_rpc_mocks={"estimatesmartfee": {"feerate": 0.00001}},  # 1 sats/vbyte
+    )
+    man = revault_network.mans()[0]
+    vaults = revault_network.fundmany([1])
+
+    # Broadcast the unvaults and get their txids
+    revault_network.activate_fresh_vaults(vaults)
+    spend_psbt = revault_network.broadcast_unvaults_anyhow(vaults, priority=True)
+
+    unvault_txids = get_unvault_txids(man, vaults)
+    spend_txid = spend_psbt.tx.hash
+
+    # Confirming the unvaults
+    bitcoind.generate_block(1, wait_for_mempool=unvault_txids)
+    for w in revault_network.participants():
+        wait_for(
+            lambda: len(w.rpc.listvaults(["unvaulted"])["vaults"]) == len(vaults),
+        )
+
+    bitcoind.generate_block(CSV - 1)
+    man.wait_for_log(f"Succesfully broadcasted Spend tx '{spend_txid}'")
+
+    for w in revault_network.participants():
+        wait_for(
+            lambda: len(w.rpc.listvaults(["spending"])["vaults"]) == len(vaults),
+        )
+
+    # Mining the spend and unvault transactions
+    bitcoind.generate_block(6)
+
+    # Manual CPFP trigger for already mined spend_txid.
+    man.rpc.cpfp([spend_txid], 50)
+    man.wait_for_log(
+        f"Nothing to CPFP in the given list.",
+    )
```

Comment:
This shouldn't be a log. Maybe the RPC command could return the list of the txids that were actually CPFPed? Or the txid of the CPFP transaction, so the caller could retrieve it, and in this case it would be `null`?